src/common/containers/TitleBar.js

```diff
@@ -112,51 +87,25 @@ class TitleBar extends Component {
     }
   }
 
-  get title() {
-    if (this.props.inGame) {
-      return (
-        <div style={{fontFamily: 'Carter One', fontSize: 32}}>
-          WORDBOTS
-        </div>
-      );
-    } else {
-      return (
-        <NavLink
-          to="/"
-          className="topLink"
-          style={{fontFamily: 'Carter One', fontSize: 32}}
-        >
-          WORDBOTS
-        </NavLink>
-      );
-    }
-  }
-
   render() {
     return (
       <div style={{height: 64}}>
         <AppBar
-          title={this.title}
+          title={
+            <div style={{fontFamily: 'Carter One', fontSize: 32}}>
+              WORDBOTS
+            </div>
+          }
           style={{
             position: 'fixed',
             top: 0
           }}
-          iconElementLeft={
-            <Tooltip
-              text={isFlagSet('sidebarCollapsed') ? 'Expand Menu' : 'Collapse Menu' }
-              place="right"
-            >
-              <IconButton onClick={this.toggleSidebar}>
-                <FontIcon className="material-icons" color="white">menu</FontIcon>
-              </IconButton>
-            </Tooltip>
-          }
+          iconElementLeft={<span />}
```

Comment:
Why is there an empty `<span />` here? If we just don't want an `iconElementLeft` in this case, it might be better to leave it unspecified and let `AppBar` decide how to display itself with no `iconElementLeft`.

Reply:
Unfortunately, `AppBar` is a third-party component that does something dumb when it isn't passed an `iconElementLeft`.